tools/continuous_batching/benchmark/continuous_batching_benchmark.cpp

```diff
@@ -507,6 +508,12 @@ int main(int argc, char* argv[]) try {
     ov::AnyMap device_config_map = {};
     if (is_speculative_decoding_enabled) {
         device_config_map.insert({ ov::genai::draft_model(draft_model_path) });
+        if (is_eagle3_mode) {
+            // disable dynamic split fuse in eagle3 mode
```

Comment:
We need to document current Eagle3 limitation or usage tips somewhere. Like lora-adapters
https://openvinotoolkit.github.io/openvino.genai/docs/guides/lora-adapters